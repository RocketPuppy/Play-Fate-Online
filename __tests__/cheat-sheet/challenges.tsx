import * as React from "react";
import * as renderer from "react-test-renderer";
import Challenges from "../../src/cheat-sheet/challenges";

test("renders all the pieces", () => {
  const render = renderer.create(<Challenges />);

  expect(render.toJSON()).toMatchSnapshot();
});
