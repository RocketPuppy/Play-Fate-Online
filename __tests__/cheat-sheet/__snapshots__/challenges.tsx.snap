// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Challenges
  </h2>
  <ul>
    <li>
      Each obstacle or goal that requires a different skill gets an overcome roll.
    </li>
    <li>
      Interpret failure, costs, and success of each roll together to determine final outcome.
    </li>
  </ul>
</div>
`;
